gpu/include/villas/gpu.hpp

```diff
@@ -68,7 +68,8 @@ class Gpu {
 
 class GpuAllocator : public BaseAllocator<GpuAllocator> {
 public:
-	static constexpr size_t GpuPageSize = 64UL << 10;
+	static constexpr
+	size_t GpuPageSize = 64UL << 10;
```

Comment:
You also merged some changes removing this style a few weeks back. I feel like we are not consistent here. I think the one line version is more readable.

Reply:
My current rule is that function/variable specifiers like

- volatile
- static
- virtual
- constexpr
- final
- inline
- friend

should be in a separate line above the return-type, name and argument list of the function.

The main motivation here is to clearly separate the two as it makes it easier/faster to read the actual return type.
Originally this convention has been introduced by Daniel, btw. :)